x-pack/plugins/reporting/server/core.ts

```diff
@@ -195,21 +242,21 @@ export class ReportingCore {
     return scopedUiSettingsService;
   }
 
-  public getSpaceId(request: KibanaRequest): string | undefined {
+  public getSpaceId(request: KibanaRequest, logger: LevelLogger): string | undefined {
```

Comment:
There are some logging changes in this PR that could be hoisted out, but I'm not sure if it will make much of a difference.